zetta_utils/mazepa_addons/configurations/execute_on_gcp_with_sqs.py

```diff
@@ -170,6 +177,8 @@ def execute_on_gcp_with_sqs(  # pylint: disable=too-many-locals
             worker_resources=worker_resources if worker_resources else {},
             ctx_managers=ctx_managers,
             worker_resource_requests=worker_resource_requests,
+            num_procs=num_procs,
+            semaphores_spec=semaphores_spec,
         )
 
     with ExitStack() as stack:
```

Comment:
Should the call below be `execute_with_pool` instead of `execute`? 
Silly github doesn't let me comment on the lines that haven't been changed